src/features/profile/pages/Profile.tsx

```diff
@@ -26,4 +49,26 @@ export const Profile: React.FC = () => {
   )
 }
 
-const Container = styled.View({ flex: 1, padding: getSpacing(5) })
+const Container = styled.View({
+  flex: 1,
+  flexDirection: 'column',
+  padding: getSpacing(5),
+})
+
+const styles = StyleSheet.create({
+  section: {
+    marginBottom: getSpacing(2),
+  },
+  row: {
+    paddingVertical: getSpacing(2),
+    marginHorizontal: getSpacing(1),
+  },
+})
+
+const Section = styled(ProfileSection).attrs({
+  style: styles.section,
+})``
+
+const Row = styled(SectionRow).attrs({
+  style: styles.row,
+})``
```

Comment:
Je loupe peut-être un truc évident ici mais ce ne serait pas plus simple comme ça ?
```suggestion
const Section = styled(ProfileSection)({
  marginBottom: getSpacing(2),
})

const Row = styled(SectionRow)({
  paddingVertical: getSpacing(2),
  marginHorizontal: getSpacing(1),
})
```